app/views/Project/AnalysisModule/index.tsx

```diff
@@ -0,0 +1,464 @@
+    '#0053af',
+    '#3f9fcf',
+    '#5fd1ef',
+];
+
+type Filter = {
+    startDate: string;
+    endDate: string;
+};
+interface AnalysisList {
+    id: number;
+    title: string;
+    createdAt: string;
+}
+
+interface AuthoringOrganizations {
+    count: number;
+    organizationTypeId: number;
+    organizationTypeTitle: string;
+}
+
+interface TimelineData {
+    key: number;
+    value: number;
+    label: string;
+}
+
+const labelSelector = (item: AuthoringOrganizations) => item.organizationTypeTitle;
+const valueSelector = (item: AuthoringOrganizations) => item.count;
+const tickFormatter = (title: string) => ({ title });
+
+const timelineLabelSelector = (item: TimelineData) => item.label;
+const timelineValueSelector = (item: TimelineData) => item.value;
+const timelineKeySelector = (item: TimelineData) => item.key;
+
+const timelineTickSelector = (d: number) => {
+    const date = new Date(d);
+    const year = date.getFullYear();
+    const month = date.getMonth();
+
+    return `${year}-${shortMonthNamesMap[month]}`;
+};
+
+interface AnalysisOverview {
+    analysisList: AnalysisList[];
+    entriesTotal: number;
+    analyzedEntriesCount: number;
+    sourcesTotal: number;
+    analyzedSourceCount: number;
+    authoringOrganizations: AuthoringOrganizations[];
+}
+
+interface AnalysisModuleProps {
+    className?: string;
+}
+
+function AnalysisModule(props: AnalysisModuleProps) {
+    const {
+        className,
+    } = props;
+
+    const {
+        project,
+    } = useContext(ProjectContext);
+    const activeProject = project?.id ? +project?.id : undefined;
+
+    const [
+        showAnalysisAddModal,
+        setModalVisible,
+        setModalHidden,
+    ] = useModalState(false);
+
+    const alert = useAlert();
+    const [iconsNode, setIconsNode] = useState<Element | null | undefined>();
+    const [actionsNode, setActionsNode] = useState<Element | null | undefined>();
+
+    const navbarContextValue = useMemo(
+        () => ({
+            iconsNode,
+            actionsNode,
+            setIconsNode,
+            setActionsNode,
+        }),
+        [iconsNode, actionsNode],
+    );
+
+    const [activePage, setActivePage] = useState(1);
+    const [analysisToEdit, setAnalysisToEdit] = useState();
+    const [dateRangeFilter, setDateRangeFilter] = useState<Filter | undefined>(undefined);
+
+    const analysisQueryOptions = useMemo(() => {
+        const endDate = dateRangeFilter?.endDate ? new Date(dateRangeFilter?.endDate) : new Date();
+        endDate.setDate(endDate.getDate() + 1);
+        // A day added to include 24 hours of endDate
+
+        return ({
+            offset: (activePage - 1) * maxItemsPerPage,
+            limit: maxItemsPerPage,
+            created_at__gte: dateRangeFilter?.startDate
+                ? getDateWithTimezone(dateRangeFilter.startDate) : undefined,
+            created_at__lt: dateRangeFilter?.endDate
+                ? getDateWithTimezone(encodeDate(endDate)) : undefined,
```

Comment:
We should slightly change the logic for created_at__lt.
IMO instead of adding 24 hours manually,
we should add this support in getDateWithTimezone

We should also ensure this behavior is consistent on DEEP.